### PR TITLE
Add OpenFAST simulation tool

### PR DESCRIPTION
**Pull request by rafmudaf**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rafmudaf** [comment]
@conda-forge-admin, please ping team
First, thanks for all your efforts on Conda-forge.

I'm able to compile OpebnFAST with `run_docker_build`, but I'm wondering where the compiled binaries should go. The `make install` command in OpenFAST is set up to place the binaries in `openfast/install/bin` by default and there are a few extra binaries that end up there. How do I tell Conda which file is the OpenFAST binary and do I need to copy that to a specific location?

I've searched through the documentation and looked at a few example recipes, but haven't found the answer. Any hints would be great.

File: recipes/openfast/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# TIFF_BIN="${PREFIX}/tiff-bin"
+# TIFF_SHARE="${PREFIX}/tiff-share"
+# TIFF_DOC="${PREFIX}/tiff-doc"
+
+# mkdir "${TIFF_BIN}" "${TIFF_SHARE}" "${TIFF_DOC}"
+mkdir build
+cd build
+
+# Pass explicit paths to the prefix for each dependency.
+cmake .. # -DCMAKE_BUILD_TYPE=RelWithDebInfo
```
Comment (chrisburr) [inline recipes/openfast/build.sh:14]:
```suggestion
cmake \
    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
    -DCMAKE_INSTALL_LIBDIR=lib \
    ..
```

File: recipes/openfast/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{% set name = "openfast" %}
+{% set version = "v2.3.0" %}
```
Comment (chrisburr) [inline recipes/openfast/meta.yaml:2]: Versions shouldn't have a v in them:
```suggestion
{% set version = "2.3.0" %}
```

File: recipes/openfast/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openfast/openfast/archive/{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/openfast/meta.yaml:9]:
```suggestion
  url: https://github.com/openfast/openfast/archive/v{{ version }}.tar.gz
```

File: recipes/openfast/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  #   - python
+  #   - make
+
+test:
+  commands:
+    - openfast -h
+
+about:
+  home: https://github.com/openfast/openfast
+  license: Apache 2.0
```
Comment (chrisburr) [inline recipes/openfast/meta.yaml:42]:
```suggestion
  license: Apache-2.0
```

File: recipes/openfast/build.sh [hunk 5]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# # Install non-lib artifacts to these special 
+# # directories so we can cleanly delete them later.
+# TIFF_BIN="${PREFIX}/tiff-bin"
+# TIFF_SHARE="${PREFIX}/tiff-share"
+# TIFF_DOC="${PREFIX}/tiff-doc"
+
+# mkdir "${TIFF_BIN}" "${TIFF_SHARE}" "${TIFF_DOC}"
```
Comment (chrisburr) [inline recipes/openfast/build.sh:9]:
```suggestion
```

File: recipes/openfast/build.sh [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+mkdir build
+cd build
+
+# Pass explicit paths to the prefix for each dependency.
```
Comment (chrisburr) [inline recipes/openfast/build.sh:13]:
```suggestion
```

File: recipes/openfast/build.sh [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    ..
+
+make -j"${CPU_COUNT}"
+
+make install
+
+# rm -rf "${TIFF_BIN}" "${TIFF_SHARE}" "${TIFF_DOC}"
+
+# For some reason --docdir is not respected above.
+# rm -rf "${PREFIX}/share/doc/tiff*"
```
Comment (chrisburr) [inline recipes/openfast/build.sh:26]:
```suggestion

```

File: recipes/openfast/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openfast/openfast/archive/v{{ version }}.tar.gz
+  sha256: 2b368e8c8211ebed03e87fb3e89ef733b7e4be4848834da4034f8419d618446c
+  # Download the tar-ball and do `openssl sha256 file.tar.gz` to update
```
Comment (chrisburr) [inline recipes/openfast/meta.yaml:11]:
```suggestion
```

File: recipes/openfast/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+build:
+  number: 0
+  skip: true  # [win]
+  # script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (chrisburr) [inline recipes/openfast/meta.yaml:16]:
```suggestion
```

File: recipes/openfast/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_cxx') }}  # [win]
+    - make
+    - cmake
+  host:
+    - libblas
+    # - libcblas
```
Comment (chrisburr) [inline recipes/openfast/meta.yaml:29]:
```suggestion
```

File: recipes/openfast/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    - liblapack
+
+  # run:
+  #   - python
+  #   - make
```
Comment (chrisburr) [inline recipes/openfast/meta.yaml:34]:
```suggestion
```

**Comment by rafmudaf** [comment]
Thanks @chrisburr. Is there any cleanup that I should add to the build script? Everything for the build is contained within the OpenFAST project directory, so is that deleted by Conda after the install?

**Comment by chrisburr** [comment]
No need to do anything. The build directory is deleted automatically if the build passes and it never affects the uploaded package.

**Pull request merged**